pseudonymisation/4-lexique.md

```diff
@@ -0,0 +1,75 @@
+---
+permalink: /pseudonymisation/lexique/
+---
+
+# Lexique des termes techniques
```

Comment:
Je n'ai pas relu ce document pour le moment

Reply:
Maintenant j'y viens !

Comme cette page est longue, je vous suggère d'utiliser une table des matières. Vous pouvez utiliser `[[toc]]` (https://v1.vuepress.vuejs.org/guide/markdown.html#table-of-contents) et vous aurez peut-être besoin de changer la profondeur de la table des matières. Il faudra avant passer les titres en `h3` par exemple